infrastructure/pyd/make_object.d

```diff
@@ -291,7 +293,7 @@ PyObject* d_string_to_python(T)(T t) if(isSomeString!T) {
     alias Unqual!(typeof(T.init[0])) C;
     static if(is(C == char)) {
         return PyUnicode_DecodeUTF8(t.ptr, cast(Py_ssize_t) t.length, null);
-    }else static if(is(C == wchar)) {
+    }else static if(is(C == wchar_t)) {
```

Comment:
I think you need to leave wchar alone in this file, otherwise conversion can't handle wstring when wchar_t isn't 16 bits.

Reply:
Yeah I figured that out after some digging, the "line 208" error by travis.ci put me on the wrong track initially. Changes in make_object.d are reverted, and commits have been adjusted for the other pull request.

Everything should be good to merge now (after travis.ci finishes).

Also: Thanks for the awesome pyd project !